src/services/classifier.ts

```diff
@@ -755,6 +760,84 @@ namespace ts {
             }
         }
 
+        function tryClassifyTripleSlashComment(start: number, width: number): boolean {
+            const tripleSlashXMLCommentRegEx = /^(\/\/\/\s*)(<)(?:(\S+)((?:[^/]|\/[^>])*)(\/>)?)?/im;
+            const attributeRegex = /(\S+)(\s*)(=)(\s*)('[^']+'|"[^"]+")/img;

```

Comment:
I think you want this to be:
`/(\S+)(\s*)(=)(\s*)('(?:[^'\\]|\\.)+'|"(?:[^"\\]|\\.)+")/img;`

Reply:
I'm having trouble diffing the regexes in my head, but I think you just added support for escaped quotation marks.  I thought about that but left it out since I believe XML uses `&quot;` to escape quotation marks.

Reply:
ahh ok.